docs/Supported-Platforms.md

```diff
@@ -105,7 +105,7 @@ Basic [smoke testing](#smoke-testing) has also validated the _oldest_
 release on which Brim appeared to work for each common distribution, as
 follows:
 
-* Ubuntu 10.04
+* Ubuntu 18.04
```

Comment:
I only just noticed this dumb typo that I've fixed here. The results at #1263 definitely don't mention testing as far back as Ubuntu 10, and the OS list at the top of the article always showed the correct version 18.04. Therefore I imagine I just made a typo back in the day, perhaps due to having "Debian 10" on the brain.